stan/math/opencl/rev/operands_and_partials.hpp

```diff
@@ -28,17 +28,10 @@ class ops_partials_edge<double, var_value<Op>,
  private:
   template <typename, typename, typename, typename, typename, typename>
   friend class stan::math::operands_and_partials;
-  const var_value<Op>& operands_;
-
-  void dump_operands(vari** varis) {}
-  void dump_partials(double* partials) {}
-  int size() { return 0; }
-  std::tuple<var_value<Op>> container_operands() {
-    return std::make_tuple(operands_);
-  }
-  std::tuple<partials_t> container_partials() {
-    return std::make_tuple(partials_);
-  }
+  var_value<Op> operands_;
+  static constexpr int size() noexcept { return 0; }
+  inline auto& operand() noexcept { return this->operands_; }
+  inline auto partial() noexcept { return this->partials_; }
```

Comment:
So I changed this to wrap this `partials_` in `to_arena()` so that when I construct the lambda for the callback in ops and partials for vars I don't need to call `to_arena()` on each partial